src/components/SearchBar.js

```diff
@@ -51,9 +51,9 @@ const SearchInput = styled.input`
 `;
 
 const NoResultsText = styled.p`
-  height: ${(props) => (props.isVisible ? 'auto' : 0)};
-  margin: ${(props) => (props.isVisible ? 'inherit' : 0)};
-  padding-top: ${(props) => (props.isVisible ? '10px' : 0)};
+  height: auto;
+  margin: inherit;
```

Comment:
I think in removing the dependency on `props.isVisible`, we could probably just remove the `height` and `margin` properties here.

Reply:
Good call, will do!